spec/updaterSpec.js

```diff
@@ -0,0 +1,167 @@
+var Updater = require('../src/updater.js');
+var Frame = require('../src/frame.js');
+var TenthFrame = require('../src/tenthFrame.js');
+
+describe("Updater", function() {
+    var perfectRoundUpToRoundNine = [[10, 0],[10, 0], [10, 0],
+                                     [10, 0], [10, 0], [10, 0],
+                                     [10, 0], [10, 0], [10, 0]];
+
+    var regularRoundUpToRoundNine = [[10, 0],[10, 0], [2, 3],
+                                     [10, 0], [10, 0], [2, 3],
+                                     [10, 0], [10, 0], [2, 3]];
+
+    var regularRoundUpToRoundNineTwo = [[10, 0],[1, 3], [3, 7],
+                                        [5, 0], [10, 0], [6, 3],
+                                        [0, 6], [1, 8], [3, 3]];
+    var partialGame = regularRoundUpToRoundNineTwo.slice(0,6);
+
+    beforeEach(function() {
+      updater = new Updater();
+      frames = [];
+      [1, 2, 3, 4, 5, 6, 7, 8, 9].forEach(function(){
```

Comment:
This is a very "ruby" way to do this. Javascript would probably use a for loop here instead:
```js
for (var i=0; i < 9; i++) {
    frames.push(new Frame());
}
```

Reply:
I actually googled something like `ruby .each in javascript` and it came back with this. I'll have to swap it out for the JS way, as this way is pretty ugly.